meetings/2023-08-16.md

```diff
@@ -0,0 +1,166 @@
+* TBL: The ETag has to do with representation, not with the body of the response, but with the state of the resource. With a PUT, the state is the body of the request. In a PATCH, the state has changed, the client needs to be able to update its ETag. The logic is always: whenever a resource state changes, the ETag changes.
+* AC: Agreed. I just want to make sure what we do does not break how browsers currently behave.
+* TBL: Browsers typically don't use a PUT.
+* AC: I mean, what if we use JavaScript, that's going to use the browser's cache.
+* TBL: It would be useful to have an overview of what we expect browsers to do.
+* SC: Sounds like a moving target. Like CORS. There is so much to do. We have a panel around QA work; we really need people. For us to show interop, we need to put the work in, either in the CG or in the WG. That includes the browser behavior: if we have any problems with how the web platform works, we can actually raise those, e.g., to the WICG.
+* RG: I want you to read [8.8.3 of RFC9110](https://httpwg.org/specs/rfc9110#field.etag)
+  > The "ETag" field in a response provides the current entity tag for the selected representation, as determined at the conclusion of handling the request. An entity tag is an opaque validator for differentiating between multiple representations of the same resource, regardless of whether those multiple representations are due to resource state changes over time, content negotiation resulting in multiple representations being valid at the same time, or both.
+* SC: I think there is indeed a nuance with the strong and weak ETags. What we currently have in the spec is what we came up with, but it is not set in stone. If you have some time, please read through them to not repeat anything. If there is new material, that might warrant a change.
+* SC: Can anyone interested add to this issue please, and then it might inform more specificity in the Solid protocol, and how much variability we expect here.
+* TBL: The protocol must give guarantees. The Solid protocol gives guarantees about the state of the resource on the Web. The ETag protocol is the best way of doing so. If it breaks, than you can lose data. We have tests, so that data will still be safe.
+
+
+
+### Chat Client-Client spec new work item
+URL: https://github.com/solid/specification/issues/553
+
+* TBL: Does anyone have an objection to the proposal?
+* SC: We are evaluating the proposal.
+* WT: My point is the one I'm making in topic below. IMO the CG and WG are not places to define domain specific interop specs.
+* SC: Why not?
+* WT: Please see discussion for next topic.
+* TBL: Client-Client specs have always been part of the design plans. We also have code in SolidOS. If they are not included in CG, we can still do it as part of Solid Project.
+* SC: It is worth reviewing the objections. My understanding is that it is within CG scope to have it as work item.
+* VB: +1
+
```

Comment:
@hzbarcea, I believe at the end of this point, you said something about not being sure whether I was properly understood, since the next issue indeed shows my belief in client-client interop within Solid. That sentence was however not recorded in the minutes, making it seem like I am indeed against any such endeavour. Do you think the following addition captures what you said correctly? If not, please suggest a better wording. Thanks!

```suggestion
* Hadrian: I think we should first look into the next topic (issue #554), because there Wouter seems to support client-client interoperability. I'm not sure how this matches what he is saying here.
```

Reply:
@csarven: This should suffice, but I leave it up to @hzbarcea whether or not to include it, since it are his words. 

Alternatively, I would propose that we include the following at the end of the topic: _"Let it be noted that, because of lack of time, Wouter did not have the opportunity to respond to this."_

Reply:
Excellent. Yes, I vaguely remember this near the end of the meeting and yes, @hzbarcea , it'd be great if you can confirm or adjust as you see fit.